cohort-parent/pom.xml

```diff
@@ -492,6 +561,47 @@
 				<artifactId>cxf-rt-transports-http</artifactId>
 				<version>3.3.8</version>
 			</dependency>
+			<dependency>
+				<groupId>com.fasterxml.jackson.core</groupId>
+				<artifactId>jackson-annotations</artifactId>
+				<version>2.11.2</version>
```

Comment:
Looks like we repeat this version a few times, so it is probably worth having a property for it.

Reply:
Thanks for calling this out, I meant to address this since we do have a jackson property that was set to something else 😬 .